python-rhsm/src/rhsm/connection.py

```diff
@@ -1405,6 +1405,20 @@ def getSubscriptionList(self, owner_key):
         results = self.conn.request_get(method)
         return results
 
+    def updateSubscriptionList(self, owner_key, auto_create_owner=False, lazy_regen=True):
+        """
+        Update subscriptions for a particular owner.
+        """
+        method = "/owners/%s/subscriptions" % self.sanitize(owner_key)
```

Comment:
By this I mean, in this method, `?` is never added to the URL, but it should be if params are added.